src/Symfony/Component/Clock/Resources/now.php

```diff
@@ -18,8 +18,12 @@
      * Note that you should prefer injecting a ClockInterface or using
      * ClockAwareTrait when possible instead of using this function.
      */
-    function now(): \DateTimeImmutable
+    function now(string $modifier = null): \DateTimeImmutable
     {
+        if (null !== $modifier) {
+            return Clock::get()->now()->modify($modifier);
```

Comment:
I think we should check the return value of `modify()` first and throw an exception if `false` is returned (and we should probably also silence the call).